### PR TITLE
Opportunity Sidebar List

### PR DESCRIPTION
**Pull request by Aimaanhasan**
Displays a sidebar that request OpportunityGetServlet to load Opportunity from the database and display it.
Worked with @waahm7 
Worked with @froheel 
Issue #35 , #49

### CODE REVIEW

**Comment by raufrajar** [comment]
Great work! It's a huge pull request though :)

**Comment by waahm7** [comment]
@raufrajar Sorry, it was supposed to be three different PR's but due to a little bit mess up of branching it ended up in one PR.

**Comment by azanbinzahid** [comment]
Is it ready to merge?

File: src/main/java/com/google/codeu/servlets/OpportunityGetServlet.java [hunk 1]
```diff
@@ -42,19 +45,134 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
...
-        response.getOutputStream().println("here");
 
-        response.getOutputStream().println(opportunity.getId());
-        response.getOutputStream().println(opportunity.getMinAge());
-        response.getOutputStream().println(opportunity.getMaxAge());
-        response.getOutputStream().println(opportunity.getTitle());
-        response.getOutputStream().println(opportunity.getDescription());
 
+        ServletOutputStream stream = response.getOutputStream();
+        stream.println("<html><head><link rel='stylesheet' href='./css/opportunity.css'></script></head> <body>");
```
Comment (azanbinzahid) [inline src/main/java/com/google/codeu/servlets/OpportunityGetServlet.java:56]: IMO whenever you have a large object in DB, it is best to keep back-end and front-end separate. Like in this servlet, a better option would be to send the opportunity in a json object, and on front-end anyone can render information according to the need.

**Pull request merged**